content/en/observability_pipelines/production_deployment_overview/_index.md

```diff
@@ -1,42 +1,188 @@
 
-Agents are configured to forward data to your Observability Pipelines Worker aggregators. Datadog recommends this role for users looking to get started with Observability Pipelines Worker because it is easy to set up and integrate into any infrastructure.
+{{< img src="observability_pipelines/production_deployment_overview/dns_service_discovery.png" alt="A diagram showing a cloud region with a cluster of agents, cluster of load balancers, and aggregate of Observability Pipelines Workers, where each group is sending separate queries to the DNS or service registry" style="width:60%;" >}}
 
- ## Reference architectures
+### Choosing protocols
 
- There are three common reference architectures that demonstrate how the two deployment roles can work in your infrastructure:
+When sending data to the Observability Pipelines Worker, Datadog recommends choosing a protocol that allows easy load-balancing and application-level delivery acknowledgment. HTTP and gRPC are preferred due to their ubiquitous nature and the amount of available tools and documentation to help operate HTTP/gRPC-based services effectively and efficiently.
 
- - **Aggregator architecture (recommended)**: Datadog recommends this architecture for new users of Observability Pipelines Worker. It deploys Observability Pipelines Worker on its own dedicated nodes for easy setup and centralized processing.
- - **Agent architecture**: This architecture deploys Observability Pipelines Worker along the edge as an agent for local data processing, taking advantage of underutilized edge resources.
- - **Unified architecture**: This architecture combines the agent and aggregator architectures. Datadog recommends evolving towards this architecture after starting with the aggregator architecture.
+Choose the source that aligns with your protocol. Each Observability Pipelines Worker source implements different protocols. For example, Observability Pipelines Worker sources and sinks use gRPC for inter-Observability Pipelines Worker communication, and the HTTP source allows you to receive data over HTTP. See [Sources][2] for their respective protocols.
+
+## Collecting data
+
+Your pipeline begins with data collection. Your services and systems generate logs, metrics, and traces that can be collected and sent downstream to your destinations. Data collection is achieved with agents, and understanding which agents to use ensures you are collecting the data you want.
+
+### Choosing agents
+
+You should choose the agent that optimizes your engineering team's ability to monitor their systems. Therefore, integrate Observability Pipelines Worker with the best agent for the job and replace the other agents with the Observability Pipelines Worker. 
```

Comment:
Delete "and replace the other agents with the Observability Pipelines Worker" and replace with: "and deploy the Observability Pipelines Worker on separate nodes as an aggregator"